ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,36 @@
-<!-- Check if the title is descriptive! -->
+<!-- Check if the title issue is descriptive! -->
 - Parent issues: -
 - Relevant issues: -
 - Estimated effort: -
 
 * * *
 
 ## User story
-The user story for the issue.
+<!-- The user story for the issue. Remove if not applicable -->
+As a...
```

Comment:
I don't like that 3 dots. If I put my cursor behind those dots and press Ctrl-backspace, the word before it (in this case "a") is deleted as well. It would be better if you remove the dots or put a space between the dots and the word before. The same applies for the rest of these documents as well